query/src/test/scala/filodb/query/exec/MetadataExecSpec.scala

```diff
@@ -434,4 +435,64 @@ class MetadataExecSpec extends AnyFunSpec with Matchers with ScalaFutures with B
       }
     }
   }
+
+  it ("should add overflow group") {
```

Comment:
A nontrivial test for a relatively simple fix -- the effort here is much appreciated